apps/challenges/aws_utils.py

```diff
@@ -531,6 +531,8 @@ def create_service_by_challenge_pk(client, challenge, client_token):
 
     queue_name = challenge.queue
     service_name = "{}_service".format(queue_name)
+    challenge.service_name = service_name
```

Comment:
@burnerlee this looks incorrect to me. The service name will be returned by the `start_workers` boto3 API call. Service name is a combination of queue name, challenge pk and a random string. Can you share how you verified that the service name is `{queue_name}_service` ?

Reply:
@burnerlee I think it wasn't clear in my message. I was asking from where did you verify that `_service` is appended at the end of the service name. If we are using it while creating a service we should create a utils method which takes challenge model object and returns service name so that there is no bug somewhere in the codebase

Reply:
As we might use service name at multiple places it would be good to have a single method which can be used to fetch service name

Reply:
oh i'll just mention that line here